### DISCUSSION TITLE
Leader key no timeout config

### DISCUSSION THREAD

**Discussion by mangkoran** [wezterm/wezterm#3196] [Q&A] [answered]
How can I make the leader key have no timeout? I followed [this config](https://wezfurlong.org/wezterm/config/keys.html#leader-key) but omitted the `timeout_milliseconds` key. However, it seems the leader key still have a "default" timeout set. I searched the documentation and found [ActivateKeyTable](https://wezfurlong.org/wezterm/config/lua/keyassignment/ActivateKeyTable.html) also have `timeout_milliseconds` key too and if I omit the key it won't deactivate (which is what I wanted with leader key).

Current workaround is to set `timeout_milliseconds` to a high value (sucs as 10000 for 10s).

**Comment by nekowinston** [comment, marked as answer]
I think your workaround is the best option currently. You could also use
```lua
leader = {
  key = "a",
  mods = "CTRL",
  timeout_milliseconds = math.maxinteger
}
```
instead of hard-coding 10000.

**Comment by pencilcheck** [comment]
I noticed that I can't chain the keybinding very fast, I have to type it very slowly before leader key is activated, this is probably not a timeout issue but activation wait, is there a way to make it shorter?
